WordPress/Classes/ViewRelated/NUX/LoginWPComViewController.swift

```diff
@@ -1,6 +1,8 @@
 import UIKit
 
 class LoginWPComViewController: SigninWPComViewController, LoginViewController {
```

Comment:
Ya, I like it because it makes it easier for the reviewer to see what's changed 😜

Otherwise the PRs have a bunch of copy & pasted code plus changed that can be hard to identify.